morevna-ep4.msrt

```diff
@@ -156,7 +156,7 @@
 32
 00:04:50,440 --> 00:04:52,580
 [rus] Прилетел огромный злой робот...
-[eng] A giantlike evil robot came...
+[eng] A giant robot came...
```

Comment:
Giantlike is not a valid English word. "Giant" can be used both as a noun and an adjective. Alternatively you can use gigantic. A child or very excited person might use ginormous. Giant and gigantic are about equally good in this context in my opinion. In fact now that I write this comment, I think I slightly prefer gigantic.

I took out evil because: do we really need to be told that the robot is evil when the next line is literally about how he murdered everyone? It really depends on what kind of personality you are trying to go for with this character.

Reply:
I also like  "gigantic". Also, I prefer to keep "evil" as this is speech-specifics for speaking character.  ^__^

Reply:
Updated